packages/ontario-frontend-cli/core/validation/validatePackageJson.js

```diff
@@ -0,0 +1,42 @@
+const logger = require('../utils/logger');
+const { readPackageJson } = require('../utils/project/readPackageJson');
+
+/**
+ * Checks if a package.json file exists in the given directory.
+ * 
+ * @param {string} dir - The directory to check for package.json.
+ * 
+ * @returns {Promise<boolean>} - Returns a promise that resolves to true if the package.json file exists, false otherwise.
+ */
+async function doesPackageJsonExist(dir = process.cwd()) {
+    try {
+      await readPackageJson(dir);
+      return true;
+    } catch (err) {
+      logger.error(`Error checking for package.json in ${dir}: ${err.message}`);
+      return false;
+    }
+  }
+
+/**
+ * Checks if the package.json file contains reference to @ongov/ontario-frontend.
+ * 
+ * @param {string} dir - The directory to check for package.json.
+ * 
+ * @returns {Promise<boolean>} - Returns a promise that resolves to true if reference to @ongov/ontario-frontend is found within the package.json file, false otherwise.
+ */
+async function isOntarioFrontendProject(dir = process.cwd()) {
+    try {
+      const packageJson = await readPackageJson(dir);
+      if (!packageJson.dependencies || !packageJson.dependencies['@ongov/ontario-frontend']) {
```

Comment:
can we make this long condition into a readable variable, ex:
```suggestion
      const hasOntarioFrontendDependency = packageJson.dependencies && packageJson.dependencies['@ongov/ontario-frontend'];
      if (!hasDependency) {

Reply:
Not opposed to this but since we are only using it in one place, not sure it's worth making a variable. To improve readability could we play with the spacing of the if statement? e.g.
```
if (
!packageJson.dependencies ||
!packageJson.dependencies['@ongov/ontario-frontend']
) {
```
I think the statement still has to be an `||` instead of an `&&`. Making it an `&&` means it'll expect `.dependencies` to be there in the `!packageJson.dependencies['@ongov/ontario-frontend']` condition but it might not always be if `.dependencies` is missing entirely.

Reply:
Very good point, thanks